python/sglang/srt/layers/attention/triton_ops/decode_attention.py

```diff
@@ -705,10 +650,10 @@ def decode_attention_fwd(
             o,
             req_to_token,
             b_req_idx,
-            b_start_loc,
```

Comment:
remove this in the func signature of `decode_attention_fwd`?

Reply:
Sure. `max_len_in_batch` and `triton_attention_reduce_in_fp32` may also need to be removed.